content/docs/user-guide/integrations/sagemaker.md

```diff
@@ -0,0 +1,138 @@
+# Amazon SageMaker
+
+## Setup
+
+Many DVC features rely on Git. To work with DVC in Amazon SageMaker, first setup
+your Git repo:
+
+1. [Clone] a repository.
+
+   ![Git clone](/img/sagemaker-git-clone.gif)
+
+2. Launch a terminal or notebook and configure the Git user name and email:
+
+   ```bash
+   git config --global user.name ...
+   git config --global user.email ...
+   ```
+
+3. Don't forget to install DVC and any other requirements in your environment!
+
+   ```bash
+   pip install dvc dvclive
+   ```
+
+## Notebooks
+
+After completing the setup, you can work with DVC in SageMaker notebooks like
+you would in any other environment. Take a look at DVC [experiments] for how to
+get started with DVC in notebooks (if you have setup [code-server] on SageMaker,
+you can also install the [DVC extension for VS Code]).
```

Comment:
@tibor-mach Added a brief mention of vs code here. Don't think we can get into any more detail here without getting too bogged down in the complexities of hosting code-server.